hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/TestDistCpOptions.java

```diff
@@ -574,4 +574,15 @@ public void testUpdateRoot() {
         .build();
     Assert.assertTrue(options.shouldUpdateRoot());
   }
+
+  @Test
+  public void testFavoredNodes() {
+    final DistCpOptions options = new DistCpOptions.Builder(
+        Collections.singletonList(
+            new Path("hdfs://localhost:8020/source")),
+        new Path("hdfs://localhost:8020/target/"))
+        .withFavoredNodes("localhost:50010")
+        .build();
+    Assert.assertNotNull(options.getFavoredNodes());
```

Comment:
prefer assertJ for new tests, with a message, and ideally verification the node value went all the way through